data/json/furniture.json

```diff
@@ -488,7 +488,7 @@
     "id": "f_sink",
     "name": "sink",
     "symbol": "&",
-    "description": "Emergency relief provider.  Water isn't running, so no water.",
+    "description": "A sink basin.  If only you could wash your hands of what you have done.  Water isn't running, so it's basically useless.",
```

Comment:
The sink includes a cabinet, not just a basin. I think you should avoid roleplaying comments here; not everyone plays a survivor that feels like they need to wash their hands of what they've done